common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
You could increase this CoA, there's a lot of place.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122671567-6609ee00-d1d8-11eb-8c8a-2ed9b6207514.png)
![image](https://user-images.githubusercontent.com/46384992/122671571-6b673880-d1d8-11eb-80de-77b5ecc5860c.png)

</details>